mitiq/interface/mitiq_qiskit/conversions.py

```diff
@@ -248,7 +248,17 @@ def from_qiskit(circuit: qiskit.QuantumCircuit) -> cirq.Circuit:
     Returns:
         Mitiq circuit representation equivalent to the input Qiskit circuit.
     """
-    return from_qasm(qasm2.dumps(circuit))
+    try:
+        mitiq_circuit = from_qasm(qasm2.dumps(circuit))
+    except Exception:
+        # If the conversion fails, try to decompose and transpile the
+        # circuit to native gates
+        circuit = compiler.transpile(
+            circuit, basis_gates=["u1", "u2", "u3", "cx"]
```

Comment:
This basis of gates was in the issue description as an excample that came from https://github.com/unitaryfund/mitiq/issues/558#issuecomment-781743206. However it requires adding some motivation on why it's the basis of choice here.

Reply:
I removed specifying the basis and let qiskit handle this to make it more general

Reply:
Based on [Qiskit docs](https://docs.quantum.ibm.com/api/qiskit/compiler#qiskit.compiler.transpile), when basis_gates is `None` (default value), the circuit is not unrolled, so I would expect that `decompose()` is what is effective here. Have you tried to print the two circuits (before and after transpile&decompose) that come out off your example?

Reply:
You're right, thank you for the feedback! Decompose is the only thing that is needed here. Transpile doesn't do anything and outputs the same circuit in this case. Decompose will decompose custom gates or packaged gates into standard qiskit gates.